src/System.Management.Automation/resources/FileSystemProviderStrings.resx

```diff
@@ -336,4 +336,7 @@
   <data name="DriveMaxSizeError" xml:space="preserve">
     <value>Maximum size for drive has been exceeded: {0}.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="ItemExists" xml:space="preserve">
+    <value>Item '{0}' already exists.</value>
```

Comment:
How about "The path {0} already exists."? That's similar to the error message when `New-Item` attempts to create a *file* that already exists, substituting "path" for "file".

I think that "Path {0} to symbolic link..." suggests that the existing item is itself a symbolic link, which is not necessarily the case.